pkg/api/dashboard.go

```diff
@@ -86,6 +86,19 @@ func DeleteDashboard(c *middleware.Context) {
 func PostDashboard(c *middleware.Context, cmd m.SaveDashboardCommand) {
```

Comment:
just an apology from me: )
sorry that PostDashboard handles Create and Update :( otherwise you could have used the nice quatas middleware here was well :(